node.gyp

```diff
@@ -344,11 +344,8 @@
             'src/inspector_socket_server.h',
           ],
           'dependencies': [
-            'deps/v8_inspector/src/inspector/inspector.gyp:standalone_inspector',
-            'v8_inspector_compress_protocol_json#host',
           ],
```

Comment:
Can you remove the dependencies list now that it's empty?

That said, I'm curious what generates `<(SHARED_INTERMEDIATE_DIR)/v8_inspector_protocol_json.h` now.  Does it perhaps work by accident because the cctest target also has a dependency on it?